apps/createsamples/utility.cpp

```diff
@@ -1270,7 +1270,7 @@ void cvCreateTrainingSamples( const char* filename,
                 if( showsamples )
                 {
                     cvShowImage( "Sample", &sample );
-                    if( cvWaitKey( 0 ) == 27 )
+                    if( (cvWaitKey( 0 ) & 0xFF) == 27 )
```

Comment:
I'm unsure whether there is a dedicated escape sequence or standard constant for ESC value. If we replace it with `'ESC'` it will be converted to (int)4543299 at least for VStudio.